Harvest.Web/ClientApp/src/Requests/ApprovalContainer.tsx

```diff
@@ -138,7 +132,7 @@ export const ApprovalContainer = () => {
                 <button className="btn btn-link mr-2">Reject</button>
```

Comment:
@srkirkland Add a disabled here to for the reject? Or wait for us to have reject code in place?